source/_docs/automation/using_blueprints.markdown

```diff
@@ -13,11 +13,11 @@ Quick links:
 
 Automations based on a blueprint only need to be configured to be used. What needs to be configured differs on each blueprint.
 
-To create your first automation based on a blueprint, go to **{% my config %}** -> **Automations & Scenes** -> **{% my blueprints %}**. Find the blueprint that you want to use and click on "Create Automation".
+To create your first automation based on a blueprint, go to **{% my blueprints title="Settings -> Automations & Scenes -> Blueprints" %}**. Find the blueprint that you want to use and click on "Create Automation".
```

Comment:
```suggestion
To create your first automation based on a blueprint, go to **{% my blueprints title="Settings > Automations & Scenes > Blueprints" %}**. Find the blueprint that you want to use and select **Create Automation**.
```
Tiny tweak. I know the -> is also quite common throughout the docs, but we're trying to use the [MS Style Guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui). It uses > in menu cascades and bold for UI elements.